libbeat/processors/add_process_metadata/gosysinfo_provider.go

```diff
@@ -52,17 +53,22 @@ func (p gosysinfoProvider) GetProcessMetadata(pid int) (result *processMetadata,
 		}
 	}
 
+	capPermitted, _ := capabilities.FromPid(capabilities.Permitted, pid)
+	capEffective, _ := capabilities.FromPid(capabilities.Effective, pid)
```

Comment:
Is there a reason why we're not checking the errors here?

Reply:
If it fails, we don't include the capabilities, same outcome if the capabilities were empty. There are valid/normal reasons for it to fail, like the process being gone by the time we try to fetch it. Also on Windows/OS X i fails with ErrUnsupported.

Reply:
Can we add a quick comment to that effect?

Reply:
Done.